src/prepare.js

```diff
@@ -94,6 +94,12 @@ function prepareElement(element, errorHandler, context) {
     return Promise.resolve([consumerFunc(value), context]);
   }
 
+  if (
+    typeof type === 'object' &&
+    type.$$typeof.toString() === 'Symbol(react.forward_ref)'
+  ) {
+    return Promise.resolve([props.children, context]);
```

Comment:
Hmm. Not sure if this will handle all the things correctly

Reply:
Agreed, this implictly ignores all descendants of a forwardref, as it never has any children. Believe this may be fixed now